src/store/state/state-model.ts

```diff
@@ -9,6 +9,8 @@ export const stateModel: StateModelIF = {
   nameRequest: {
     nrNumber: '',
     entityType: '',
+    details: {},
+    applicant: {},
```

Comment:
I'm not sure if it makes any difference here, but you could initialize the data as
`applicant: {} as NameRequestApplicantIF ,`

Thoughts?

Reply:
Done. Good to know the interfaces at a glance.